R/analyze.R

```diff
@@ -134,9 +134,9 @@ set_sign <- function(x) {
 }
 
 confint_relative_get <- function(timings, branches, reference, ci) {
-  no_change <- "&nbsp;&nbsp;:ballot_box_with_check:"
+  no_change <- ":heavy_check_mark:"
```

Comment:
nit: Why the switch from :ballot_box_with_check: to :heavy_check_mark:? Aesthetic preference or is there a readability advantage or something like that?

Reply:
Purely aesthetic - not sure why but on Chromium this gets rendered to a small thin rather than heavy checkmark:

<img width="400" alt="image" src="https://github.com/lorenzwalthert/touchstone/assets/1156307/26b1e1eb-1cb4-402f-8abd-4cde8eb4d9b7">

But I don't feel strongly - happy to revert in fact.

Reply:
Ah I don't mind it. I might even prefer it? I was mainly asking because there is sometimes stuff that's better for not obvious but important reasons like screen-reader friendliness or something :)
My version ^^:
![image](https://github.com/lorenzwalthert/touchstone/assets/16141871/c10cbf98-0803-417d-b9c3-8b925516b66a)